src/objects/database.lzz

```diff
@@ -122,6 +140,126 @@ public:
 	}
 
 private:
+	class UpdateHook {
+	public:
+
+		explicit UpdateHook(
+			Addon * addon,
+			v8::Isolate * isolate,
+			Database * db,
+			v8::Local<v8::Function> fn
+		) :
+			addon(addon),
+			isolate(isolate),
+			db(db),
+			fn(isolate, fn) {}
+
+		static void invoke(void * data, int op, char const * dbName, char const * tableName, sqlite3_int64 rowid) {
+			UpdateHook * self = static_cast<UpdateHook *>(data);
+
+			v8::Isolate * isolate = self->isolate;
+			v8::Isolate::Scope isolateScope(isolate);
+			v8::HandleScope handleScope(isolate);
+
+			v8::Local<v8::Context> context = isolate->GetCurrentContext();
+
+			v8::Local<v8::Value> opArg = self->addon->cs.Op(isolate, op);
+			v8::Local<v8::Value> dbNameArg = v8::String::NewFromUtf8(isolate, dbName, v8::NewStringType::kNormal).ToLocalChecked(); 
+			v8::Local<v8::Value> tableNameArg = v8::String::NewFromUtf8(isolate, tableName, v8::NewStringType::kNormal).ToLocalChecked(); 
+			v8::Local<v8::Value> rowidArg = v8::BigInt::New(isolate, rowid);
```

Comment:
It might be more appropriate to use `v8::Number` if `safeIntegers` is not enabled, as that would be consistent with the rest of the library's behavior. I think it should accept a `safeIntegers` option similarly to how `db.function()` works.